akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -789,7 +790,9 @@ class Http2ServerSpec extends AkkaSpec with WithInPendingUntilFixed with Eventua
 
     def expectDecodedResponseHEADERS(streamId: Int, endStream: Boolean = true): HttpResponse = {
       val headerBlockBytes = expectHeaderBlock(streamId, endStream)
-      decodeHeadersToResponse(headerBlockBytes)
+      val decoded = decodeHeadersToResponse(headerBlockBytes)
+      // filter date to make it easier to test
+      decoded.withHeaders(decoded.headers.filterNot(_.is("date")))
```

Comment:
Yep, good that will simplify writing tests. Will now probably also need to exclude server as the default configuration should have one set.